odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -239,6 +239,33 @@ def test_dont_write_the_existing_childs(self):
         a = parent.child_ids[0]
         parent.write({'child_ids': [Command.link(a.id), Command.create({'name': 'B'})]})
 
+    def test_create_with_commands(self):
+        child1 = self.env['test_new_api.order.line'].create({'product': 'set'})
+        child2 = self.env['test_new_api.order.line'].create({'product': 'sept'})
+
+        # warm up default_value
+        self.env['test_new_api.order'].create({
+            'line_ids': [Command.set(child1.ids)],
+        })
+
+        # INSERT, UPDATE of child1
+        with self.assertQueryCount(2):
+            self.env['test_new_api.order'].create({
+                'line_ids': [Command.set(child1.ids)],
+            })
+
+        # INSERT parent
+        # INSERT thief_parent, UPDATE of child1/child2
```

Comment:
Using "parent" and "child" is confusing, as the relation is not a hierarchy.  Simply use `order` and `line`.